sdk/mixedreality/azure-mixedreality-remoterendering/src/main/java/com/azure/mixedreality/remoterendering/models/RenderingSession.java

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.mixedreality.remoterendering.models;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+
+/** Holds the properties of a rendering session. */
+public final class RenderingSession {
```

Comment:
Add `@Immutable` annotations to model classes that have only getter methods. Also, add `@Fluent` annotation to model classes that have setter methods that return `this`.